app/models/intake.rb

```diff
@@ -576,15 +575,36 @@ def formatted_mailing_address
     ADDRESS
   end
 
-  def create_intake_document(filename)
-    filename ||= "2020 13614-C with 15080.pdf"
+  def update_or_create_13614c_document(filename)
     pdf_tempfile = pdf
     pdf_tempfile.seek(0)
-    client.documents.create!(document_type: DocumentTypes::Form13614CForm15080.key, intake: self, upload: {
-      io: pdf_tempfile,
-      filename: filename,
-      content_type: "application/pdf",
-      identify: false
-    })
+    document = client.documents.find_or_initialize_by(document_type: DocumentTypes::Form13614CForm15080.key)
+    document.update!(
+      document_type: DocumentTypes::Form13614CForm15080.key,
+      intake: self,
```

Comment:
I actually think it is safe to remove the reference to intake on these documents -- it's kind of a vestige of when we were connecting documents to the intake instead of the client last season. Not necessary to do now, but could be a good clean up task.